app/src/main/java/org/oppia/android/app/options/OptionsActivity.kt

```diff
@@ -42,7 +42,7 @@ class OptionsActivity :
       isFromNavigationDrawer: Boolean
     ): Intent {
       val intent = Intent(context, OptionsActivity::class.java)
-      intent.putExtra(KEY_NAVIGATION_PROFILE_ID, profileId)
+      intent.putExtra(NAVIGATION_PROFILE_ID_ARGUMENT_KEY, profileId)
```

Comment:
Per #1587 in cases when the key is being used for an activity extra rather than a fragment argument, we should name it ``EXTRA`` rather than ``ARGUMENT``.

I think overall in this PR you may need to duplicate the key in a few places. I suggest having a unique version of this key for each activity & fragment that requires it as part of their intents & arguments, respectively, rather than trying to reuse a single key in lots of places. This is a change from the current structure, but it's more correct & better fits the intended style of #1587.

Reply:
Also as a follow-up: the 'navigation' prefix doesn't make as much sense in most of these locations, so having separate variables with better contextualized names will help (e.g. ``OPTIONS_PROFILE_ID_EXTRA_KEY`` is a lot clearer).